render/expected/expected.go

```diff
@@ -17,18 +17,21 @@ var (
 		ID:                unknownPseudoNode1ID,
 		LabelMajor:        "10.10.10.10",
 		Pseudo:            true,
+		NodeMetadata:      report.MakeNodeMetadata(map[string]string{}),
```

Comment:
Ugh. This breaks down in the app tests, when we unmarshal to a RenderableNode, which has an invisible (`json:"-"`) NodeMetadata field. We'll have to nil-out the expected NodeMetadatas in that instance. I hate reference semantics.